keras-runner/src/main/kotlin/org/jetbrains/research/keras/runner/deserializer/json/ActivationType.kt

```diff
@@ -0,0 +1,9 @@
+package org.jetbrains.research.keras.runner.deserializer.json
+
+import kotlinx.serialization.*
+
+@Serializable
+enum class ActivationType {
+    @SerialName("relu") ReLU,
```

Comment:
Let's use here just real names (like `relu`) instead of `@SerialName` everywhere